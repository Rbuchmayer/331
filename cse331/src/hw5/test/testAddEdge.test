CreateGraph g
AddNode g n1
AddNode g n2
AddNode g n3
AddNode g n4
AddEdge g n1 n1 e1
AddEdge g n1 n2 e2
AddEdge g n3 n1 e3
AddEdge g n3 n1 e3
AddEdge g n1 n4 e4
AddEdge g n4 n4 e1
AddEdge g n4 n4 e1