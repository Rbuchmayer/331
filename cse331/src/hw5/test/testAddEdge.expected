created graph g
added node n1 to g 
added node n2 to g
added node n3 to g
added node n4 to g
added edge e1 from n1 to n1 in g
added edge e2 from n1 to n2 in g
added edge e3 from n3 to n1 in g
added edge e3 from n3 to n1 in g
added edge e4 from n1 to n4 in g
added edge e1 from n4 to n4 in g
added edge e1 from n4 to n4 in g